packages/dashboard-v2/src/pages/settings/index.js

```diff
@@ -26,51 +34,38 @@ const AccountPage = () => {
             </p>
           </section>
           <hr />
-          {!isLargeScreen && (
-            <section>
-              <AvatarUploader className="flex flex-col sm:flex-row gap-8 items-center" />
-            </section>
-          )}
           <section className="flex flex-col gap-8">
-            <FormGroup>
-              <TextInputBasic label="Display name" placeholder="John Doe" />
-              <div className="flex mt-2 sm:mt-0 justify-center">
-                <Button>Update</Button>
-              </div>
-            </FormGroup>
-            <FormGroup>
-              <TextInputBasic label="Email" placeholder="john.doe@example.com" />
-              <div className="flex mt-2 sm:mt-0 justify-center">
-                <Button>Update</Button>
-              </div>
-            </FormGroup>
-            <FormGroup>
-              <TextInputBasic type="password" label="Password" placeholder="dbf3htf*efh4pcy@PXB" />
-              <div className="flex mt-2 sm:mt-0 justify-center order-last sm:order-none">
-                <Button>Update</Button>
-              </div>
-              <small className="text-palette-400">
-                The password must be at least 6 characters long. Significantly different from the email and old
-                password.
-              </small>
-            </FormGroup>
+            {state === State.Failure && (
+              <Alert $variant="error">There was an error processing your request. Please try again later.</Alert>
+            )}
+            {state === State.Success && <Alert $variant="success">Changes saved successfully.</Alert>}
+            <AccountSettingsForm
+              user={user}
+              onSuccess={() => {
+                reloadUser();
```

Comment:
do we want to `await` reload ?

Reply:
It won't hurt :) 2aa3437a

Reply:
we probably want to also `catch` potential bad reload since otherwise we'll never change to success state

Reply:
Fair point. Should even feel a bit snappier now: 5bb1ed18.